ads/_config.js

```diff
@@ -534,6 +534,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'jubna': {},
```

Comment:
If you have implemented `renderStart` by @lannka suggetsion, you will want to add `renderStartImplemented: true` here 😄

Reply:
No, I didn't use renderStart while implementation.